package.json

```diff
@@ -82,6 +82,7 @@
     "yeoman-generator": "^1.0.1"
   },
   "devDependencies": {
+    "@polymer/tools-common": "^1.0.1",
```

Comment:
Follow up to tsconfig.extends chain below: 

I still don't feel 100% comfortable with the `^` meaning that our build config can be changed remotely, especially if the tools-common dependency is used/versioned for more than just tsconfig. How do you feel about pinning to explicit version here?